cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -85,7 +85,7 @@ func (dc *DownstreamController) syncPod(stop chan struct{}) {
 				klog.Infof("send message successfully, operation: %s, resource: %s", msg.GetOperation(), msg.GetResource())
 			}
 		case <-stop:
-			klog.Info("stop syncPod")
+			klog.Error("stop syncPod")
```

Comment:
Should not make the log level error since SIG_TERM received will enter this case.

Reply:
done